### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by trevorhardy**
### Summary

If merged this pull request will update the documentation to remove/correct references to the Fundamental Final example.

### Proposed changes

Update documentation TOCs and links

### CODE REVIEW

File: docs/build_RTD.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+python -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html
```
Comment (nightlark) [inline docs/build_RTD.sh:1]: Fixes the shellcheck error about not knowing what the shell syntax to lint against.

```suggestion
#!/bin/sh
python -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html
```

**Review by phlptp — APPROVED** [review]

File: docs/build_RTD.sh (hunk 1 above)
Comment (nightlark) [reply docs/build_RTD.sh:1]: After this precommit fix is made the PR can be merged. It could also be /bin/bash instead.

**Review by nightlark — CHANGES_REQUESTED** [review]

**Review by trevorhardy — COMMENTED** [review]
All looks good to me. @nightlark, ready for your approval, I think.

**Review by nightlark — APPROVED** [review]

**Comment by trevorhardy** [comment]
@nightlark or @phlptp, the "Merge pull request" button on Github is grayed out for me with a corresponding message "Required status must pass before merging." Is there something else I need to do to merge this?

**Comment by nightlark** [comment]
@trevorhardy it's just one of the intermittent CI tests failing... just need to rerun the failing required check until it passes

**Pull request merged**